ui/src/app/views/workspace-edit.tsx

```diff
@@ -348,6 +386,9 @@ export interface WorkspaceEditState {
   loading: boolean;
   showUnderservedPopulationDetails: boolean;
   showStigmatizationDetails: boolean;
+  autocompleteDiseases: Array<string>;
+  diseaseHover: Array<boolean>;
+  searchTermChangedEvent: Function;
```

Comment:
This state variable is unused. In the constructor, the searchTermChangedEvent debounced function is assigned directly as an instance property (which is fine, IMO since it's not involved in the rendering of the component).